toothpick-kotlin/README.md

```diff
@@ -0,0 +1,115 @@
+# Toothpick Extensions  
+Toothpick Extensions
+
+### Base Extensions
+```kt
+// Inject from Scope
+scope.inject(obj)
+
+// Inject with Kotlin Class
+scope.getInstance(KotlinClass::class)
+// Inject with reified type
+val type: KotlinClass = scope.getInstance()
+
+// In a Module, bind with Kotlin Class
+module.bind(KotlinClass::class)...
+
+// Inject dependency without specifying type
```

Comment:
the type ?

Reply:
Without passing the class (e.g., `Dependency::class`), type is the wrong word here.